instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import './PostContainer.css'
+import { library } from '@fortawesome/fontawesome-svg-core'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import heart from '../SearchBar/heart.png'
+import speech from './speech.png'
+import { faIgloo } from '@fortawesome/free-solid-svg-icons'
+import CommentSection from '../CommentSection/CommentSection';
+import moment from 'moment';
+library.add(faIgloo);
```

Comment:
I see you're using the library for font awesome - great job looking into it! It's pretty powerful but I've never used it in any of my projects